test/tests/server.js

```diff
@@ -377,4 +377,49 @@ describe('Server', () => {
         });
     });
   });
+
+  function writeToDiskWithMultiCompiler(value, done) {
```

Comment:
maintainer's preference here: please do shorten this function name. something under 15 characters preferably, e.g. `multiToDisk`